optuna_dashboard/_cached_extra_study_property.py

```diff
@@ -65,11 +72,15 @@ def update(self, trials: List[FrozenTrial]) -> None:
             if trial.state not in states_of_interest:
                 continue
 
+            if len(trial.intermediate_values) > 0:
```

Comment:
[nits] How about changing here like following to reduce a bit of computation since we don't need to update `self.has_intermediate_values` if it's already True?

```suggestion
            if self.has_intermediate_values or len(trial.intermediate_values) > 0:
```

Reply:
Thank you very much.
When ` self.has_intermediate_values` is True , there is no need to run L76, so
Isn't this place `if not self.has_intermediate_values and len(trial.intermediate_values) > 0:`

Reply:
Oh, sorry. My suggestion is incorrect.

We can actually skip the evaluation for `len(trial.intermediate_values) > 0` if making the condition of if statement like `not self.has_intermediate_values or len(trial.intermediate_values) > 0`.